<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>价格 (元/s):</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>静态功耗 (nW):</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="staticPowerLineEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="computingPowerLineEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="computingTypeComboBox">
     <item>
      <property name="text">
       <string>请选择...</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>IP地址</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>存储空间 (GB):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="maskLineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>开关电容 (fF):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>子网掩码：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="switchingCapacitanceLineEdit"/>
   </item>
   <item row="10" column="2">
    <widget class="QPushButton" name="confirmButton">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>计算能力 (FLOPS):</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="priceLineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="ipLineEdit"/>
   </item>
   <item row="10" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="storageSpaceLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>算力类型 (CPU/GPU):</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>混合能源参数：</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="powermixLineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
